### PR TITLE
Enhance NodeJS Integration Tests - wait for nodejs process to be terminated

### PR DESCRIPTION
**Pull request by psrna**

### CODE REVIEW

**Comment by psrna** [comment]
Triggered jst.itests jenkins job twice to confirm it's working: https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/jst.it.smoke/jdk=jdk1.8,label=rhel7/48/console

File: tests/org.jboss.tools.jst.ui.bot.test/src/org/jboss/tools/jst/ui/bot/test/nodejs/NodeJSLauncherTest.java [hunk 1]
```diff
@@ -29,74 +28,102 @@
...
+        ConsoleView console = new ConsoleView();
+        console.activate();
+        new WaitUntil(new ConsoleHasText("Debugger listening"), TimePeriod.LONG);
+        
+        assertTrue("Node.js App is not debugging!", console.getConsoleText().contains("Debugger listening"));
+        
+        terminate();
+    }
+    
+    private void terminate(){
```
Comment (rawagner) [inline tests/org.jboss.tools.jst.ui.bot.test/src/org/jboss/tools/jst/ui/bot/test/nodejs/NodeJSLauncherTest.java]: please consider making this "@after" method. That way even if some test fails you can be sure that all appropriate processes were terminated.
If you put @after here then dont forget to make it more robus (ie getNodeJSLaunch can be null, or launchListener can be null too etc)

Comment (psrna) [inline tests/org.jboss.tools.jst.ui.bot.test/src/org/jboss/tools/jst/ui/bot/test/nodejs/NodeJSLauncherTest.java]: good point. thx!

**Comment by psrna** [comment]
@rawagner fixed. Please review. Thx!

**Pull request merged**